app/models/micropost.rb

```diff
@@ -1,6 +1,9 @@
 class Micropost < ApplicationRecord
   belongs_to :user
-  scope :feed, ->(id){where "user_id = ?",id}
+  scope :feed, lambda{|user|
+    where("user_id IN (:following_ids) OR user_id = :user_id",
```

Comment:
scope :feed, ->(user) do
    where("user_id IN (:following_ids) OR user_id = :user_id",
      user_id: user.id, following_ids: user.following_ids)
  end
em cũng sửa lại như thế này rồi, có được không c